components/org.wso2.carbon.identity.authz.valve/src/main/java/org/wso2/carbon/identity/authz/valve/util/Utils.java

```diff
@@ -66,4 +69,31 @@ public static boolean isUserBelongsToRequestedTenant(AuthenticationContext authe
         }
         return tenantDomainFromURLMapping.equals(tenantDomain);
     }
+
+    /**
+     * Checks whether cross-tenant domain is allowed for the tenant.
```

Comment:
`WDYT of something like this Check whether the tenant domain is allowed as a cross-tenant domain`?